src/components/LogRow/BaseRow/index.tsx

```diff
@@ -142,8 +143,9 @@ const ProcessedBaseRow: React.FC<ProcessedBaseRowProps> = memo((props) => {
     let render = children;
     if (searchTerm) {
       // escape the matching string to prevent XSS
-      render = render.replace(
+      render = highlighter(
```

Comment:
Might be something funky going on with the replace regex, `Script` should be red and `deploy` should be purple but they are both a bright yellow not defined in the LeafyGreen system

<img width="1405" alt="Screen Shot 2023-02-13 at 5 48 48 PM" src="https://user-images.githubusercontent.com/47064971/218591979-697e0846-e4e5-44dd-b7a2-3cf0017144e0.png">

Reply:
Looks like the color was being passed in as undefined the yellow is the default color for the `<mark> tag </mark>`